### DISCUSSION TITLE
(Feature request] ROUTER direct message echo

### DISCUSSION THREAD

**Discussion by zl3ag** [meshtastic/firmware#5070] [Ideas]
Idea: Any direct message sent to a node set as ROUTER is echoed back to the sender as a direct message.

This would be very handy when testing things like messaging apps and also when demoing Meshtastic to people. Much more exciting for them to see a message being replied to rather than sending one out and getting ignored completely.

**Comment by dirspb** [comment]
Doesn't traceroute do it?
